src/plugins/legacy/itkDataSHImage/management/vtkSphericalHarmonicSource.cxx

```diff
@@ -253,11 +254,14 @@ void vtkSphericalHarmonicSource::PrintSelf(ostream& os,vtkIndent indent) {
        << NumberOfSphericalHarmonics << std::endl;
     os << indent << "SH Basis: " << NumberOfSphericalHarmonics << "x"
        << sphereT->GetOutput()->GetNumberOfPoints() << std::endl;
+
     os << indent << "SH Coefficients:" << std::endl << '[';
     for (int i=0;i<NumberOfSphericalHarmonics;++i)
+    {
         os << indent << SphericalHarmonics[i] << " ";
-        os << indent << ']' << std::endl;
```

Comment:
Olivier are you agree with this change?

Reply:
(my compiler does not like the indentation, it asked for clarity, that's why the "for" loop has now brackets)

Reply:
it should do:  [number0 number1 ...]